docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -119,3 +181,50 @@ The important part is having a WordPress site installed, and know where and how
 Alternative editors include [Sublime Text](https://www.sublimetext.com/) that is also available across platforms, though is a commercial product; or other free alternatives include [Vim](https://www.vim.org/), [Atom](https://atom.io/), and [Notepad++](https://notepad-plus-plus.org/) all support standard JavaScript style development.
 
 You can use any editor you're comfortable with, it is more a personal preference. The development setup for WordPress block editor is a common JavaScript environment and most editors have plugins and suppport. The key is having a way to open, edit, and save text files.
+
+## Uninstall - Start Over
+
+Here are a few instructions if you need to start over, or want to remove what was installed.
+
+### Local Environment
+
+* If you just want to reset and clean the WordPress database:
+
+```
+wp-env clean all
+```
+
+* To remove the local environment completely for a specific project:
+
+```
+wp-env destroy
+```
+
+* To completely uninstall wp-env tool:
+
+```
+npm -g uninstall @wordpress/env
+```
+
+* To uninstall Docker, or Visual Studio Code use your OS method to remove packages. For example, on Windows run "Add or remove programs". You can additionally uninstall from the Docker Desktop app, click the bug icon at the top to switch to this Troubleshoot screen. Click Uninstall or remove.
+
+![Docker Troubleshoot Screenshot](https://developer.wordpress.org/files/2020/08/docker-uninstall-screen.png)
+
+
+### Uninstall Node/NVM
+
+To uninstall Node/NVM, delete the NVM directory, this is typically installed at `$HOME/.nvm`, delete using
+
+```
+rm -rf "$HOME/.nvm"
+```
+
+If this does not work and the `$NVM_DIR` environment variable is set you can remove using `rm -rf "$NVM_DIR"`
+
+To clean up any installed JavaScript packages remove the global `.npm` directory at `$HOME/.npm`,
+
+```
+rm -rf "$HOME/.npm"
+```
+
+Just as you confirmed the installation worked, you can confirm the uninstall worked if you try to run `npm -v`, `node -v`, and `nvm -v` you should then see a "command not found" error in the terminal.
```

Comment:
Adding in that one needs to quit and restart terminal.

Rephrasing:
Just as you confirmed the installation worked, you can confirm the uninstall worked. First quit and restart terminal and then try to run `npm -v`, `node -v`, and `nvm -v` you should then see a "command not found" error in the terminal.